tests/tests/swfs/avm2/net_navigateToURL/Test.as

```diff
@@ -0,0 +1,34 @@
+package {
+    import flash.net.URLRequest;
+	import flash.net.URLVariables;
+	import flash.net.navigateToURL;
+	import flash.display.MovieClip;
+	public class Test extends MovieClip {
+        public function Test()
+        {
+            var request:URLRequest = new URLRequest("https://example.com/purchase/");
+			var variables:URLVariables = new URLVariables();
+            var TEST_PARAM_NAME:String = "sku";
+            var cases:Array = [
+                ["POST", "Test"],
+                ["POST", "sku=Test"],
```

Comment:
I think just doing `["POST", "Test"], ["POST, variables], ...` would be move obvious.

Reply:
Fixed in https://github.com/ruffle-rs/ruffle/pull/15129/commits/89da73ada2f1474c3179bea0ed321ae76b842f7f